pygatt/bled112_bglib.py

```diff
@@ -0,0 +1,1732 @@
+"""
+MODIFIED Bluegiga BGAPI/BGLib implementation
```

Comment:
I added MODIFIED to some of the files that I modified that had copyright stuff on them. I still have to talk to Peplin about the copyright information